flutter_keyboard_visibility/pubspec.yaml

```diff
@@ -4,17 +4,39 @@ version: 6.0.0
 homepage: https://github.com/MisterJimson/flutter_keyboard_visibility
 repository: https://github.com/MisterJimson/flutter_keyboard_visibility
 
+publish_to: 'none' # TODO: remove on fork merge
+
 environment:
   sdk: '>=2.12.0 <4.0.0'
   flutter: ">=1.20.0"
 
 dependencies:
   meta: ">=1.0.0 <2.0.0"
-  flutter_keyboard_visibility_platform_interface: ^2.0.0
-  flutter_keyboard_visibility_linux: ^1.0.0
-  flutter_keyboard_visibility_macos: ^1.0.0
-  flutter_keyboard_visibility_web: ^2.0.0
-  flutter_keyboard_visibility_windows: ^1.0.0
+  flutter_keyboard_visibility_platform_interface:
+    git:
```

Comment:
the reviewers have the ability to test it via current working pubspec dependencies.
After I get an approval for this PR, I will revert pubspec changes

Reply:
@MisterJimson 
also, if you are satisfied with general idea, I will update all places of the doc comments across the package to the `status` instead of `isKeyboardVisible`